### PR TITLE
added repository status labels

### PR DESCRIPTION
**Pull request by tldavies**

### CODE REVIEW

File: docs/repositories.rst [hunk 1]
```diff
@@ -78,3 +78,26 @@ Documentation
      - Description
    * - https://github.com/RackHD/docs
      - The RackHD documentation as published to http://rackhd.readthedocs.org/en/latest/.
+
+
+Repositories Status
```
Comment (benbp) [inline docs/repositories.rst]: "Repository statuses" ?

**Comment by benbp** [comment]
"Repositories status" reads a little weird to me but I don't have an alternative.

**Comment by benbp** [comment]
:+1:

File: docs/repositories.rst [hunk 2]
```diff
@@ -78,3 +78,26 @@ Documentation
...
+^^^^^^^^^^^^^^^^^
```
Comment (yyscamper) [inline docs/repositories.rst]: What does this mean?

Comment (benbp) [inline docs/repositories.rst]: I believe it shows live code coverage/build status?

Comment (heckj) [inline docs/repositories.rst]: yep, these are the "tags and badges" that we have enabled on the README's for each of the repos - and show build status (travisCI), test coverage (coveralls), static analysisresults (code climate)

**Comment by heckj** [comment]
+1

**Comment by heckj** [comment]
@tldavies this seems cut short - did you mean to add the other repositories with their build statii as well?

**Comment by benbp** [comment]
Yeah, wouldn't it be better to just wait until we have all of them and merge in one PR rather than splitting it up?

**Comment by tldavies** [comment]
@benbp yes! Here you go :-)

**Comment by benbp** [comment]
Thanks. I updated the PR description.

**Comment by bossidy** [comment]
really cool!

Most of repos ALTed to "Test Coverage". Do they need to be badged? (Not that I know what 'badged' means.)

**Comment by bossidy** [comment]
I noticed that the table is taking a a while to populate. I wonder if we need a note telling the user to wait while the table loads...

**Comment by tldavies** [comment]
@bossidy It looks like the Code Coverage link only works through the github version and not through the plan docs. I will create a new PR to fix. Not sure about the loading time, as it really depends but we can add a note that it could a bit, so please wait.

**Pull request merged**